tools/intl_remap_demo.py

```
@@ -0,0 +1,352 @@
+        push(held(prompt_view('PICKLTR', tgt), letter_pos[src]), "Press", src, args.flash)
+        assign[tgt] = src
+        push(held(intl_view(), intl_pos), f"{tgt} now hosts", src, args.pick_hold)
+
+        # give it its own accent with the picker, or all three would show the same one
+        row = row_for(tgt)
+        if ctrl_pos is not None and acc < len(row):
+            push(held(held(intl_view(), intl_pos), ctrl_pos), "Tap", "Ctrl", args.flash)
+            pages = page_count(len(row), args.slots)
+            armed = picker_row(held(held(intl_view(), intl_pos), ctrl_pos), row, 0, pages)
+            sel = dict(armed)
+            c = copy.copy(sel[target_pos(tgt)]); c.selected = True
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)